.gitlab-ci.yml

```diff
@@ -1150,13 +1150,13 @@ test_full_integration:
       junit: results_full_integration.xml
 
 # Publish acceptance test coverage into coveralls when either running tests for a
-# mender PR (MENDER_REV != "master") or new merge into mender/master that requires
+# mender PR (MENDER_REV ~= "pull/XXX/head") or new merge into mender/master that requires
 # Docker images publishing (PUBLISH_DOCKER_CLIENT_IMAGES == "true")
 .template_publish_acceptance_coverage:
   only:
     variables:
       - $PUBLISH_DOCKER_CLIENT_IMAGES == "true"
-      - $MENDER_REV != "master"
+      - $MENDER_REV =~ "/pull\/.*\/head/"
```

Comment:
Question:
Why do you escape some of the slashes only?

Reply:
The slashes are the delimiters for the regular expression. See https://docs.gitlab.com/ee/ci/yaml/#common-if-clauses-for-rules

Actually, the quote should not be there (I'll fix that)